README.md

```diff
@@ -1 +1,46 @@
-# Cr-Portfolio_Project
\ No newline at end of file
+# curriculum-cr-training-contact-form
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
```

Comment:
[Optional]
It would be good practice if you create a live version of your project to be viewed from your readme file.